devnets/startNetwork.ts

```diff
@@ -216,3 +220,15 @@ async function spawnNode(tempDir: string, binary: string, args: string[], signal
     }
   })
 }
+
+function addXcmHrmpChannels(genesisConfig: GenesisConfig, paraIds: number[]) {
+  if (!genesisConfig.hrmp) {
+    genesisConfig.hrmp = { preopenHrmpChannels: [] }
+  }
+  for (const senderParaId of paraIds) {
+    for (const recipientParaId of paraIds) {
+      if (senderParaId === recipientParaId) continue
+      genesisConfig.hrmp.preopenHrmpChannels.push([senderParaId, recipientParaId, 8, 512])
```

Comment:
Can the 8 and 512 constants be extracted? (Should they be configurable?)

Reply:
I'll update the function signature and set these values as default.

Given that these values can be different for each permutation, the alternative would be to code each permutation in the `capi.config.ts`

@tjjfvi  WDYT?

Reply:
What resulting behavior does this impact? Is it meaningful to JS devs?

Reply:
I guess that we could see an impact if there are many messages, but given that this is for dev, I think we should be fine with these values